plugins/scaffolder-backend/api-report.md

```diff
@@ -369,7 +376,7 @@ export function fetchContents({
   reader: UrlReader;
   integrations: ScmIntegrations;
   baseUrl?: string;
-  fetchUrl?: JsonValue;
+  fetchUrl?: string;
```

Comment:
Interested if this is technically a breaking change, even though there was a check inside the function to make sure it was a string, and this is a narrowing of the type?

Reply:
Nah, not a breaking change imo, let's call it a bug fix. Seeing as it could never work at runtime anyway